Assignment2/q2.js

```diff
@@ -0,0 +1,38 @@
+let a = [5,2,3,6,9,8,7,1];
+console.log("Original Array - ", a);
+console.log("Type of a = ", typeof a);
+a.push(13);
+a.push(55);
+console.log("Array after adding 13 and 55 - ", a);
+a.pop();
+console.log("Array after deleting last element - ",a);
+console.log("Sorted array - ",a.sort())
+
+const b = [50,65,48,21,15,35];
+console.log("Original array - ",b);
```

Comment:
Will you please add output after each console

Reply:
ok I will

Reply:
First, make the changes commit and push 
then resolve the conversation with a comment

Reply:
All changes were made as per your instructions.